### DISCUSSION TITLE
Images as blocks (only)

### DISCUSSION THREAD

**Discussion by whawker** [remirror/remirror#704] [Ideas]
Hi there.

I'm new to Remirror, and I've been playing around with the image extension.

From what I can gather it seems that the default behaviour for images is to be inline, however I'd like to restrict images to be blocks instead. I can't see any obvious way to do this, other than creating my own extension?

I imagine this might be difficult with the draggable behaviour? By making sure it can only exist between other blocks?

Thanks

**Comment by ifiokjr** [comment]
@whawker I've added this as an issue #707
